packages/components/tag/src/tag-body.tsx

```diff
@@ -63,7 +63,9 @@ const getContentWrapperStyles = (props: TTagBodyProps) => {
     border-style: solid;
     border-width: 1px;
     border-color: ${props.type === 'warning'
-      ? designTokens.colorWarning
+      ? designTokens.colorWarning85
+      : props.isDisabled
```

Comment:
I think the disabled state should override both `warning` and `normal` types.
![image](https://github.com/commercetools/ui-kit/assets/97907068/88ca708f-92dc-46b7-9c8a-cf1ec409314b)

@FilPob Could you please confirm? 🙏

Reply:
Yes, the disabled state should be same for both `Default` and `warning` type. @CarlosCortizasCT  good catch, thanks

Reply:
Sorry for the missing values... I added the missing styles **[here](https://www.figma.com/file/PaImaWorWz1RbV85gqw4MH/Filter-%26-Table-controls?type=design&node-id=1133%3A127328&mode=design&t=00bf6LyKQTa9dmS5-1)**

I added:
- Warning tag in the hover state: bg-color: `color-warning-85`

> This is nbd, but just to confirm.. on the warning default, do you want the border color to match the neutral of default disabled - or do you want it to remain orange (with neutral background color)?

@valoriecarli  the warning tag in the default state should have border-color: `color-warning-85`. Is that what you asked?